rust/protocol/src/curve/curve25519.rs

```diff
@@ -133,16 +191,24 @@ impl KeyPair {
         bool::from(cap_r_check.as_bytes().ct_eq(&cap_r))
     }
 
+    /// Return a public key that another party can use to validate against [Self::verify_signature].
```

Comment:
…or calculate a key agreement. I think this is just "Returns the public key in the pair."